assets/src/scripts/monitoring-location/components/hydrograph/index.js

```diff
@@ -1,43 +1,35 @@
-/**
+/*
```

Comment:
This module has change significantly. It might be easier to view the file stand alone.